provider/testdata/data/json/datetime/symbols@1/japanese/bn.json

```diff
@@ -0,0 +1,852 @@
+{
+  "months": {
+    "format": {
+      "abbreviated": [
+        "জানু",
+        "ফেব",
+        "মার্চ",
+        "এপ্রিল",
+        "মে",
+        "জুন",
+        "জুলাই",
+        "আগস্ট",
+        "সেপ্টেম্বর",
+        "অক্টোবর",
+        "নভেম্বর",
+        "ডিসেম্বর"
+      ],
+      "narrow": [
+        "জা",
+        "ফে",
+        "মা",
+        "এ",
+        "মে",
+        "জুন",
+        "জু",
+        "আ",
+        "সে",
+        "অ",
+        "ন",
+        "ডি"
+      ],
+      "short": null,
+      "wide": [
+        "জানুয়ারী",
+        "ফেব্রুয়ারী",
+        "মার্চ",
+        "এপ্রিল",
+        "মে",
+        "জুন",
+        "জুলাই",
+        "আগস্ট",
+        "সেপ্টেম্বর",
+        "অক্টোবর",
+        "নভেম্বর",
+        "ডিসেম্বর"
+      ]
+    },
+    "stand_alone": {
+      "abbreviated": [
+        "জানুয়ারী",
+        "ফেব্রুয়ারী",
+        "মার্চ",
+        "এপ্রিল",
+        "মে",
+        "জুন",
+        "জুলাই",
+        "আগস্ট",
+        "সেপ্টেম্বর",
+        "অক্টোবর",
+        "নভেম্বর",
+        "ডিসেম্বর"
+      ],
+      "narrow": null,
+      "short": null,
+      "wide": null
+    }
+  },
+  "weekdays": {
+    "format": {
+      "abbreviated": [
+        "রবি",
+        "সোম",
+        "মঙ্গল",
+        "বুধ",
+        "বৃহস্পতি",
+        "শুক্র",
+        "শনি"
+      ],
+      "narrow": [
+        "র",
+        "সো",
+        "ম",
+        "বু",
+        "বৃ",
+        "শু",
+        "শ"
+      ],
+      "short": [
+        "রঃ",
+        "সোঃ",
+        "মঃ",
+        "বুঃ",
+        "বৃঃ",
+        "শুঃ",
+        "শনি"
+      ],
+      "wide": [
+        "রবিবার",
+        "সোমবার",
+        "মঙ্গলবার",
+        "বুধবার",
+        "বৃহস্পতিবার",
+        "শুক্রবার",
+        "শনিবার"
+      ]
+    },
+    "stand_alone": null
+  },
+  "day_periods": {
+    "format": {
+      "abbreviated": {
+        "am": "AM",
+        "pm": "PM",
+        "noon": null,
+        "midnight": null
+      },
+      "narrow": {
+        "am": "AM",
+        "pm": "PM",
+        "noon": null,
+        "midnight": null
+      },
+      "short": null,
+      "wide": {
+        "am": "AM",
+        "pm": "PM",
+        "noon": null,
+        "midnight": null
+      }
+    },
+    "stand_alone": null
+  },
+  "eras": {
+    "names": {
```

Comment:
Issue: including the premodern era names bloats data size. I recall us discussing that premodern eras would use Gregorian eras.

Reply:
I don't recall that being resolved, perhaps we should have a followup? I think the resolution was that premodern eras get the more complex era codes.

Reply:
I think we may consider shipping the historic eras as an optional extension. But yes, we should file an issue, because this is simply way too much data without a great way to slice it

Reply:
Yeah we can have `calendar/japanese@1/{modern, historical}` variants and by default it only loads modern. An interesting question would be if it should load historical unconditionally if it's present in the data, or if it should be an option during construction (`enum Historical { Yes, No, Try }`)

A nice property of such an architecture is that we can have the hardcoded fallback for when modern data is unavailable as well.

Reply:
This feels like a good mentorable help wanted bug

Reply:
https://github.com/unicode-org/icu4x/issues/1409